projects/sbb-esta/angular-business/links/src/link/link.component.ts

```diff
@@ -0,0 +1,48 @@
+/** Autogenerated by public2business schematic. DO NOT CHANGE! */
```

Comment:
Why do you commit autogenerated files? Or is it modified and the comment ist still here?

Reply:
I think you have commited the files accidentally, so please remove them :)

Reply:
Weird how these got committed. I removed them.